src/main/scala/beam/sim/BeamSim.scala

```diff
@@ -474,16 +474,17 @@ class BeamSim @Inject() (
       }
     }
 
-    if (beamConfig.beam.physsim.skipPhysSim) {
-      Await.result(Future.sequence(List(outputGraphsFuture)), Duration.Inf)
+    val physsimFuture = if (beamConfig.beam.physsim.skipPhysSim) {
```

Comment:
As this is only being waited on, there is no need for the first condition block:

```
val physsimFuture = Future{
  if (!beamConfig.beam.physsim.skipPhysSim){
    agentSimToPhysSimPlanConverter.startPhysSim(event, initialTravelTime.orNull)
  }
}
```